ml-agents/mlagents/trainers/stats.py

```diff
@@ -0,0 +1,109 @@
+from collections import defaultdict
```

Comment:
Seems like while you have updated tests to reflect the usage of the classes in this module, we aren't explicitly testing the functionality captured here.  Could you add some tests (you should just be able to mock out the `tf.summary.FileWriter`)?

Reply:
Added test